### ISSUE TITLE
A11y_trace.playwright.dev_Select file's_Screen reader: As soon as the "Could not load trace from..." error is displayed, the narrator does not announce that information.

### ISSUE THREAD

**Issue by SachinChavhan317** [microsoft/playwright#29095]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Actual Result:
As soon as the "Could not load trace from..." error is displayed, the narrator does not announce that information.

**Note:** Issue is reproducing with NVDA.

### Expected Result:
As soon as the "Could not load trace from..." error is displayed, the screen reader should announce that information.

### GitHub Tags:
#A11yMAS;#A11yTCS;#Playwright releases;#P3_trace.playwright.dev_Web_Jan2024;#A11ySev2;#AILimited;#ChromiumEdge;#DesktopWeb;#FTP;#WCAG1.3.1;#Info and Relationships;#Narrator;#Win11;#P3_WCP;

### Environmental Details:
Application Name: trace.playwright.dev
#URL: https://trace.playwright.dev/

### OS & Browser Details:
Windows 11 Enterprise Insider Preview
OS build	26025.1000
Microsoft Edge
Version 122.0.2348.0 (Official build) dev (64-bit)
Chrome Version: Version 120.0.6099.225 (Official Build) (64-bit)
Narrator
NVDA Version: 2023.3

### Repro Steps:

1. Hit on this URL: https://trace.playwright.dev 
2. "Drop Playwright Trace to load" page will appear.
3. Turn on Screen reader
4.  TAB to "Select a file(s)" button and select wrong file.
5. Verify whether as soon as the "Could not load trace from..." error is displayed, the narrator is announcing that information or not.

### User Impact:
Screen reader users will not get about the "Could not load trace from..." error is displayed if screen reader does not announce it.

### Attachment:
Narrator -
![Narrator1](https://github.com/microsoft/playwright/assets/86241494/ceb1f55a-f66f-452a-80ec-580ba3a56f77)

NVDA -
![NVDA1](https://github.com/microsoft/playwright/assets/86241494/87679b37-8d5d-4194-9377-dfd6f054bc49)

**Comment by mxschmitt** [comment]
This one should be fixed on our "beta version" which is deployed here: https://trace.playwright.dev/next/

Next production release is in 4-6 weeks.

**Comment by Raghu-addepalli** [comment]
@mxschmitt verified the issue in below environment issue is still reproducing with NVDA and attachments below.

 https://trace.playwright.dev/next/


https://github.com/microsoft/playwright/assets/91586743/3af23461-4c1c-46b4-a0fe-5043e4906905


https://github.com/microsoft/playwright/assets/91586743/e65c2bf5-f0ae-4978-9f3a-b9368915eb18

**Comment by mxschmitt** [comment]
This should be fixed, please re-check.

**Comment by Raghu-addepalli** [comment]
verified the issue in below environment issue is Fixed hence closing the bug and added attachment below.
https://trace.playwright.dev/next/

![Uploading 29095_Fixed.png…]()
